components/description/description.module.scss

```diff
@@ -0,0 +1,55 @@
+$colors: #456c3d, #669f54, #7ec16a, #b9e7ac, #ecedf0;
+
+.squareContainer li {
+  width: 3vmin;
+  height: 3vmin;
+  border-radius: 5px;
+  margin: 15px 0;
+  transform: rotate(-90deg);
+}
+
+@for $i from 1 through length($colors) {
+  .squareContainer li:nth-of-type(#{$i}) {
+    background: darken(nth($colors, $i), 0%);
+    border: 5px solid darken(nth($colors, $i), 5%);
+    box-shadow: 0 0 5px 1px lighten(nth($colors, $i), 25%);
+  }
+}
+
+.container {
+  width: 100%;
+  min-height: 45vh;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 4vmin;
+  font-weight: 300;
+  font-family: Assistant;
+  background-color: white;
+  box-shadow: 0 0 10px 5px rgb(0, 0, 0, 0.2);
+  padding: 25px;
+  flex-wrap: wrap;
+  position: relative;
+  margin-bottom: -30px;
+
+  @media only screen and (max-width: 1286px) {
+    .squareContainer {
+      transform: rotate(90deg);
+      margin: 0 auto;
+    }
+  }
+}
+
+.brandColor {
+  color: #ce5a55;
+}
```

Comment:
Please remove and get the color from the colors file, https://github.com/urish/pull-request-community/blob/master/styles/colors.ts